Formula/modsman.rb

```diff
@@ -0,0 +1,21 @@
+class Modsman < Formula
+  desc "Minecraft mod manager for the command-line"
+  homepage "https://gitlab.com/sargunv-mc-mods/modsman"
+  url "https://gitlab.com/sargunv-mc-mods/modsman.git",
+      :tag      => "0.20.1"
+  sha256 "09834f6b34014652e6dd46a4e3c0ecb9d353002d44053854d05faffd3efc022b"
+
+  depends_on :java => "1.8+"
+
+  def install
+    system "./gradlew", ":modsman-cli:installDist"
```

Comment:
This will download dependencies right? Because that's not acceptable for homebrew-core.

Reply:
This was modeled after other formulas like this one: https://github.com/Homebrew/homebrew-core/blob/master/Formula/golo.rb

How do you recommend I build this without downloading dependencies?

Reply:
That one only pulls in dependencies on a head build, it ships them with the release for normal builds.

Reply:
Ah, so I'll need to set up my CI to build and upload the dist tarball, and then modify this formula to download install that tarball instead of building from source (in non-head builds). Can do. Would that satisfy the requirement?

Reply:
Yeah, as long as the normal install doesn't go around pulling "random" files from the internet it's fine. (https://docs.brew.sh/Acceptable-Formulae#we-dont-like-install-scripts-that-download-unversioned-things)